#include "team_client_player_handler.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ClientPlayerTeamServiceHandler::TeamInfoNotify(entt::entity player,const ::TeamInfoS2CRequest* request,
	     TeamInfoS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

